args/physics/colliders/convexcollider.hpp

```diff
@@ -150,6 +160,7 @@ namespace args::physics
 
             HalfEdgeFace* eghf = new HalfEdgeFace(eg, math::vec3(0, 1, 0));
             halfEdgeFaces.push_back(eghf);
+            eghf->id = " eghf";
```

Comment:
Either write a comment explaining that these are edges combined into a face, or make more descriptive names.
Maybe ASCII art 3D cube in the summary?(glyns idea)

Reply:
![image](https://user-images.githubusercontent.com/37859544/95436152-ce1ee980-0953-11eb-950a-133f1945434b.png)

This is already done at the beginning of the function

Reply:
I'm of the opinion that I'd still like more descriptive names, no matter how tedious.
However, that is what glyn requested for more description so I approve.